README.md

```diff
@@ -0,0 +1,138 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  
+
+  <h3><b>Hello Microverse Project README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Hello Microverse\] ](#-hello-microverse-)
+  - [🛠 Built With **HTML and CSS**](#-built-with-html-and-css)
+    - [Tech Stack **HTML and CSS**](#tech-stack-html-and-css)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+**Hello Microverse** is a project under microverse Module 1 block ! aimed at testing my knowledge about the concepts learnt in the first 2 days of the program.
+
+## 🛠 Built With **HTML and CSS**
+
+### Tech Stack **HTML and CSS**
+### Key Features <a name="key-features"></a>
+- **Displays Hello Microverse**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: A web browser.
+### Setup
+
+Clone this repository to your desired folder:
+cd Module_1
+-git clone git@github.com:As1imwe-Mark/HelloWorld.git
+
+### Install
+
+Install this project with:
+_No installation is required, double click the index.html file to open it.
+
+### Usage
+
+To run the project, execute the following command:
+-Double click the index.html file.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Asiimwe Mark**
+
+- GitHub: @As1imwe-Mark
+- Twitter:@markasiimwe
+- LinkedIn:AsiimweMark
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```

Comment:
- [ ] The last reviewer pointed out that you remove this section which serves as a comment/ description to guide you. Kindly do so to make your work look clean and proffestuonal 👍